spec/components/schemas/datetime.yaml

```diff
@@ -1,4 +1,4 @@
 type: string
 format: datetime
-description: ISO 8601 date time with timezone
```

Comment:
I really want to keep the timezone - I think it is very important.

Reply:
oh - I missed that this was a new schema.

Reply:
I think it would be preferable to only have one date-time format. My preference is for date-times to contain the timezone.

Reply:
Have gone back to one datetime schema, with timezone included